src/patient-search/patient-search.component.tsx

```diff
@@ -1,10 +1,35 @@
 import React from "react";
 import { match } from "react-router";
+import styles from "./patient-search.component.css";
 
 export default function PatientSearch(props: PatientSearchProps) {
-  return <section>The patient search!</section>;
+  return (
+    <div>
+      <div className={styles.patientSearchHeader}>
+        <button onClick={props.history.goBack} className={styles.iconBtn}>
+          <svg className="omrs-icon" fill="var(--omrs-color-interaction)">
+            <use xlinkHref="#omrs-icon-arrow-back" />
```

Comment:
I disagree with the home app taking over the full screen and having a back button to get back to the main app with the primary navbar.

I’m working with JJ and Greg on this to find out how we can proceed.